npm/xpath/.releaserc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  ...require('../../.releaserc.base'),
+  branches: [
+    { name: 'master', channel: 'latest' },
```

Comment:
```suggestion
    { name: 'develop', channel: 'latest' },
```
Since https://github.com/cypress-io/cypress/pull/23899 merged, this needs updated